.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+# The ignored dependencies are so that we get open PRs that result proper nested changelogs
+
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Isn't daily too frequent? We might get too much noise if it is daily, weekly might be better. Also, is dependabot something we decided to use company-wide or just on the sdk?

Reply:
We are using it in JF and Lucian pointed me to their config, which I then used as basis for this. We have also extended VB to detect the change type from the title prefix, which this config instructs dependabot to add. So I was just thinking whether we could have this a try in the SDK. Still haven't decided though.

Reply:
@thgreasi yeah we can give it a go I guess, but I still think daily might be too much :) I'll leave it up to you to decide, and feel free to self-approve.